@arunsathiya : Thank you for your patience 🙇 In kicking off the the testing workflows, it appears we have several tests failing [here](https://github.com/cli/cli/actions/runs/6356109172/job/17314248142?pr=8024#step:6:126)

```shell
Error: required flag(s) "id" not set
Usage:
  item-edit [flags]

Examples:
# edit an item's text field value
gh project item-edit --id <item-ID> --field-id <field-ID> --project-id <project-ID> --text "new text"

# clear an item's field value
gh project item-edit --id <item-ID> --field-id <field-ID> --project-id <project-ID> --clear


Flags:
      --body string                      Body of the draft issue item
      --clear                            Remove field value
      --date string                      Date value for the field (YYYY-MM-DD)
      --field-id string                  ID of the field to update
      --format string                    Output format: {json}
  -h, --help                             help for item-edit
      --id string                        ID of the item to edit
      --iteration-id string              ID of the iteration value to set on the field
      --number float32                   Number value for the field
      --project-id string                ID of the project to which the field belongs to
      --single-select-option-id string   ID of the single select option value to set on the field
      --text string                      Text value for the field
      --title string                     Title of the draft issue item

Error: only one of `--text`, `--number`, `--date`, `--single-select-option-id` or `--iteration-id` may be used
Usage:
  item-edit [flags]

Examples:
# edit an item's text field value
gh project item-edit --id <item-ID> --field-id <field-ID> --project-id <project-ID> --text "new text"

# clear an item's field value
gh project item-edit --id <item-ID> --field-id <field-ID> --project-id <project-ID> --clear


Flags:
      --body string                      Body of the draft issue item
      --clear                            Remove field value
      --date string                      Date value for the field (YYYY-MM-DD)
      --field-id string                  ID of the field to update
      --format string                    Output format: {json}
  -h, --help                             help for item-edit
      --id string                        ID of the item to edit
      --iteration-id string              ID of the iteration value to set on the field
      --number float32                   Number value for the field
      --project-id string                ID of the project to which the field belongs to
      --single-select-option-id string   ID of the single select option value to set on the field
      --text string                      Text value for the field
      --title string                     Title of the draft issue item

--- FAIL: TestRunItemEdit_Draft (0.00s)
    item_edit_test.go:196: 
        	Error Trace:	/home/runner/work/cli/cli/pkg/cmd/project/item-edit/item_edit_test.go:196
        	Error:      	Received unexpected error:
        	            	SilentError
        	Test:       	TestRunItemEdit_Draft
--- FAIL: TestRunItemEdit_Text (0.01s)
    item_edit_test.go:246: 
        	Error Trace:	/home/runner/work/cli/cli/pkg/cmd/project/item-edit/item_edit_test.go:246
        	Error:      	Received unexpected error:
        	            	SilentError
        	Test:       	TestRunItemEdit_Text
--- FAIL: TestRunItemEdit_Number (0.01s)
    item_edit_test.go:295: 
        	Error Trace:	/home/runner/work/cli/cli/pkg/cmd/project/item-edit/item_edit_test.go:295
        	Error:      	Received unexpected error:
        	            	SilentError
        	Test:       	TestRunItemEdit_Number
--- FAIL: TestRunItemEdit_Date (0.01s)
    item_edit_test.go:345: 
        	Error Trace:	/home/runner/work/cli/cli/pkg/cmd/project/item-edit/item_edit_test.go:345
```

Starting to look over the failures and what about the code changes that is a problem

--- a/pkg/cmd/project/item-edit/item_edit.go
+++ b/pkg/cmd/project/item-edit/item_edit.go
@@ -138,23 +138,17 @@ func NewCmdEditItem(f *cmdutil.Factory, runF func(config editItemConfig) error) 
 func runEditItem(config editItemConfig) error {
 	// when clear flag is used, remove value set to the corresponding field ID
 	if config.opts.clear {
-		if err := clearItemFieldValue(config); err != nil {
-			return err
-		}
+		return clearItemFieldValue(config)
 	}
 
 	// update draft issue
 	if config.opts.title != "" || config.opts.body != "" {
-		if err := updateDraftIssue(config); err != nil {
-			return err
-		}
+		return updateDraftIssue(config)
 	}
 
 	// update item values
 	if config.opts.text != "" || config.opts.number != 0 || config.opts.date != "" || config.opts.singleSelectOptionID != "" || config.opts.iterationID != "" {
-		if err := updateItemValues(config); err != nil {
-			return err
-		}
+		return updateItemValues(config)
 	}
 
 	if _, err := fmt.Fprintln(config.io.ErrOut, "error: no changes to make"); err != nil {

